src/cli/cli_controller.py

```diff
@@ -0,0 +1,40 @@
+"""
+This module contains all logic pertaining to the CLI commands
+"""
+
+from config.config_controller import read_config_file
+from helpers.json_helper import write_json_file, read_json_file
+
+def handle_cli_args(cli_args: list[str]) -> None:
+    """
+    This function will take redirect to the corresponding function for the CLI arg.
+
+    Args:
+        cli_args (list[str]): The CLI args passed.   
+    """
+    if len(cli_args) < 1:
+        print("This should be the help command.")
+    elif cli_args[0] == "create":
+        adding_content()
+
+def adding_content():
+    """
+    This function will handle adding a task to the tasks.json file
+    """
+    config = read_config_file()
```

Comment:
Line 24-28 and Line 40

Could we create tasks_helper.py for reading and writing to tasks.json file? 

I am thinking this because we probably will be reading and writing to the tasks.json file in several functions in the future, and having one/two functions to do this instead of having to rewrite above code will just make our lives a bit easier, what do you think?